smartlearner/tasks/tests/test_stopping_criteria.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
```

Comment:
There are two conventions: either we put a tests module along side every other modules or we reproduce the entire folders structure of the library in a folder `tests/` at the root.